tests/testthat/test-ggplot-coord.R

```diff
@@ -0,0 +1,52 @@
+context("Fixed coordinates")
+
+# Expect trace function
+expect_traces <- function(gg, n_traces, name) {
+  stopifnot(is.ggplot(gg))
+  stopifnot(is.numeric(n.traces))
+  save_outputs(gg, paste0("coord_fixed-", name))
+  L <- gg2list(gg)
+  all_traces <- L$data
+  no_data <- sapply(all_traces, function(tr) {
+    is.null(tr[["x"]]) && is.null(tr[["y"]])
+  })
+  has_data <- all_traces[!no_data]
+  expect_equal(length(has_data), n_traces)
+  list(traces = has_data, layout = L$layout)
+}
+
+# Data where x ranges from 0-10, y ranges from 0-30
+set.seed(202)
+dat <- data.frame(xval = runif(40,0,10), yval = runif(40,0,30))
```

Comment:
I remember that we decided not to use randomized data in our test suite, so that the image diff's remain meaningful.  But I may have totally missed a discussion on the topic in the meantime...  If so, please let me know.